src/signals/incident/components/form/index.ts

```diff
@@ -38,7 +37,6 @@ export default {
   HandlingMessage,
   HiddenInput,
   LinkButton,
-  MapInput,
   MapSelect,
```

Comment:
is it possible to remove the MapInput component as well?

Reply:
No, it's used in the incident detail page.